src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2613,4 +2616,65 @@ namespace winrt::TerminalApp::implementation
                    winrt::hstring{ fmt::format(L"<{}>", RS_(L"UnnamedWindowName")) } :
                    _WindowName;
     }
+
+    // Method Description:
+    // - Called when an attempt to rename the window has failed. This will open
```

Comment:
Curious why this one goes out or comes back in through applogic? Page is originating the Rename Request Args, and it can check Succeeded. What's the flow graph look like for getting here?

Reply:
I suppose that's a good point. It's definitely bubbling all the way up, then back down (after a pair of `co_await`s)